src/components/ThankYouResponse.tsx

```diff
@@ -0,0 +1,16 @@
+import { h } from 'preact';
+
+type Props = {
+  title: string;
+  subtext: string;
+};
+
+export const ThankYouResponse = ({ title, subtext }: Props) => {
+  return (
+    <div className={'text-center px-8 min-w-[489px]'}>
+      <p className={'text-5xl'}>❤️</p>
+      <p className={'text-2xl py-2 font-semibold'}>{title}</p>
```

Comment:
Can we component for `Question/Title` and `Description/Subtext`. We'll need them in other components